sdk/network/Azure.ResourceManager.Network/src/Custom/VirtualMachineScaleSetExtensionResource.cs

```diff
@@ -0,0 +1,125 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.ResourceManager.Network
+{
+    /// <summary>
+    /// Extensions for VirtualMachineScaleSet.
+    /// </summary>
+    public class VirtualMachineScaleSetExtensionResource : ArmResource
```

Comment:
according to [this comment](https://github.com/Azure/azure-sdk-for-net/issues/28185#issuecomment-1116659864) this should rename to `VirtualMachineScaleSetNetworkResource`